README.md

```diff
@@ -13,9 +13,9 @@ Detailed instructions are available in our Installation Guides below.
 Sumo Logic Helm Chart Version
 | version | status |  
 |--|--|
-|[0.17.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.17.0/deploy/README.md)| current / supported  |
-|[0.16.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.16.0/deploy/README.md) | deprecated |
-|[0.15.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.15.0/deploy/README.md) | deprecated |
+|[0.17.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v0.17/deploy/README.md)| current / supported  |
```

Comment:
@perk-sumo @breinero I think we should also document the fact that only the last 2 helm chart versions will be supported so that customers are aware of it and do timely upgrades.

Reply:
Yeah, that's a fair point, we should add it - could you please add a jira ticket (or just create a PR?) so that we don't loose it?